database/postgresql/postgresql_test.go

```diff
@@ -40,7 +40,7 @@ func (suite *DbTestSuite) SetupTest() {
 		dbCfg,
 		codec,
 		logging.DefaultLogger(),
-		types.DefaultAddressParser(),
+		nil,
```

Comment:
If this isn't used, do we need still to pass it to the database constructor? Can't we remove it as a field from the database entirely?

Reply:
This is null just becouse is not used from our implementation but we don't know what other people will implement.

Reply:
If anyone wants to extend the `Database` implementation and have it a codec field, they can do it by defining their own implementation and wrapping Juno's `Database` one: 

```go
type Database struct {
  junodb.Database
  cdc codec.Codec
}
```

We shouldn't have fields that we are not using